addons/website/views/snippets/s_cta_card.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_card" name="Card Call to Action">
+    <section class="s_cta_card o_cc o_cc2 pt64 pb64" data-oe-shape-data="{'shape':'web_editor/Blocks/03'}">
+        <div class="o_we_shape o_web_editor_Blocks_03"/>
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-9">
+                    <h2>50,000+ companies run Odoo to grow their businesses.</h2>
+                    <p class="lead">Join us and make your company a better place.</p>
+                </div>
+                <div class="col-lg-3">
+                    <div class="s_card card o_cc o_cc1 ms-auto shadow" data-name="Card" style="border-width: 0;" data-vcss="001">
```

Comment:
Set the `max-width` to `50%`. This will ensure a good visual result even at very high resolution.
Note that the **minimum** width is `min(18rem/100%)` by default and this rule takes priority, meaning that the card will never be too small on mobile 👍
Also, the max-width value is editable, thus we're covered even if the user needs a bigger card 😉

```suggestion
                    <div class="s_card card o_cc o_cc1 ms-auto shadow" data-name="Card" style="border-width: 0;  max-width: 50%;" data-vcss="001">
```